arenadata/Dockerfile.ubuntu

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04 as base
+FROM ubuntu:22.04 AS base
```

Comment:
How does this (and below) change relate to the current patch?

Reply:
Eliminate warnings

Reply:
I think this should have been done in a separate patch.

Reply:
What kind of warnings are we talking about? I don't get any warnings from docker during the build.

Reply:
> What kind of warnings are we talking about? I don't get any warnings from docker during the build.

#1 WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)

Reply:
I still think this should been fixed in a separate patch.

Reply:
> in a separate patch

I think this only creates unneeded friction. Why such small changes need that much effort?